source-salesforce/tests/conftest.py

```diff
@@ -19,14 +19,14 @@ def time_sleep_mock(mocker):
 
 @pytest.fixture(scope="module")
 def bulk_catalog():
-    with open("unit_tests/bulk_catalog.json") as f:
+    with open("source-salesforce/tests/bulk_catalog.json") as f:
```

Comment:
does this scoping change if you add an empty `__init__.py` to this tests/ folder ?

Reply:
The `__init__.py` is already  at the folder =P